server/routes/UserRoute.js

```diff
@@ -381,4 +380,18 @@ router.get('/userclubs/:id', async (req, res) => {
   }
 });
 
+// @desc    User  get all current user book clubs
+// @route   GET /user/userclubs/:id
+// @access  Private
+router.post('/bookclubs', async (req, res) => {
```

Comment:
Looks like the @route needs to be fixed. Though I personally think that was the better way to do it, considering you are getting data and not posting it.

Reply:
Will fix it